modules/navier_stokes/test/tests/ins/jacobian_test/jacobian_stabilized_test.i

```diff
@@ -0,0 +1,117 @@
+# This input file tests the jacobians of many of the INS kernels
+[GlobalParams]
+  gravity = '1.1 1.1 1.1'
+  u = vel_x
+  v = vel_y
+  w = vel_z
+  p = p
+  integrate_p_by_parts = true
+  laplace = true
+  stabilize = true
+  alpha = 1.1
```

Comment:
It may be worth having a comment about what the different global parameters do since this is a test. Without digging through the classes, I assume `stabilize = true` turns on SUPG stabilization, so a more specific name like `supg = true` might be more appropriate now that we have multiple forms of stabilization possible.

Reply:
`_stabilize` is actually a parameter in the `INSBase` class. If set to true it applies SUPG stabilization for momentum residuals and PSPG stabilization for the mass residual. I could remove `_stabilize` from that class and make `_pspg` in `INSMass` and `_supg` in `INSMomentumBase` if that's perhaps more transparent to the users.

Reply:
> If set to true it applies SUPG stabilization for momentum residuals and PSPG stabilization for the mass residual.

I thought I saw a `mooseError` that gets thrown if you try to turn on SUPG when the convective term is off? I think there should be a way of independently enabling/disabling the two different types of stabilization so we can handle the Stokes flow case (PSPG=on, SUPG=off), right?